dhcpv6/dhcpv6relay.go

```diff
@@ -180,3 +181,53 @@ func (r *DHCPv6Relay) GetInnerPeerAddr() (net.IP, error) {
 	}
 	return addr, nil
 }
+
+// NewRelayReplFromRelayForw creates a RELAY_REPLY packet based on a RELAY_FORW
```

Comment:
should be RELAY_REPL (because it was cheaper than RELAY_REPLY in the RFC :-(   )

Reply:
right! :)